src/deviceOauthService.ts

```diff
@@ -7,9 +7,9 @@
 /* eslint-disable camelcase */
 /* eslint-disable @typescript-eslint/ban-types */
 
-import Transport from 'jsforce/lib/transport';
+import Transport from '@jsforce/jsforce-node/lib/transport';
```

Comment:
Can we change this to some publicly exported Class/Function in jsforce-node?  If not, do we need to make a change in jsforce-node?